### ISSUE TITLE
Laptop mode incorrectly set to False

### ISSUE THREAD

**Issue by javiribera** [lucasdiedrich/gnome-display-switcher#15]
In the "Display Switcher" settings, there is a text that says
"Laptop mode: False"
while I am using a laptop.

I am using GNOME Shell 3.18.5

Here is the output of `xrandr`:

```
Screen 0: minimum 8 x 8, current 3286 x 1200, maximum 32767 x 32767
LVDS1 connected 1366x768+0+0 (normal left inverted right x axis y axis) 344mm x 194mm
   1366x768      60.02*+
   1360x768      59.80    59.96  
   1280x720      60.00  
   1024x768      60.00  
   1024x576      60.00  
   960x540       60.00  
   800x600       60.32    56.25  
   864x486       60.00  
   640x480       59.94  
   720x405       60.00  
   680x384       60.00  
   640x360       60.00  
DP1 disconnected (normal left inverted right x axis y axis)
HDMI1 connected primary 1920x1200+1366+0 (normal left inverted right x axis y axis) 518mm x 324mm
   1920x1200     59.95*+
   1920x1080     60.00    50.00    59.94    30.00    25.00    24.00    29.97    23.98  
   1920x1080i    60.00    50.00    59.94  
   1600x1200     60.00  
   1280x1024     75.02    60.02  
   1152x864      75.00  
   1280x720      60.00    50.00    59.94  
   1024x768      75.08    60.00  
   800x600       75.00    60.32  
   720x576       50.00  
   720x576i      50.00  
   720x480       60.00    59.94  
   720x480i      60.00    59.94  
   640x480       75.00    60.00    59.94  
   720x400       70.08  
VGA1 disconnected (normal left inverted right x axis y axis)
VIRTUAL1 disconnected (normal left inverted right x axis y axis)
```

As stated by `xrandr`, there is a 1920x1200 HDMI screen connected to the laptop. Maybe it is related with this issue is that my primary and secondary screens are flipped. When I select primary screen, the HDMI screen is used, and the "Mirror" option gets the resolution from the big, HDMI screen, trying to fit 1920x1200 pixels into my built-in laptop display.

Besides fixing the broken autodetection of the laptop mode, maybe it is wise to let the user manually change it if necessary.

**Comment by lucasdiedrich** [comment]
@javiribera, we could break this in two differents issue, one is, laptop mode is not working properly because we don't "support" LVDS, and the second one is let the user choose which which mode he wan'ts to use in case the suggested one don't work for him.

I can make both i think, the problem now is that i'm out of time, maybe next week i can make with the rest of the issues.

Thanks for the issue.

**Comment by lucasdiedrich** [comment]
This extension will be disabled since this functionalty is available at Gnome 3.26+. To use Gnome just press 'Super' + P and hold Super.
